Gemfile

```diff
@@ -53,7 +53,7 @@ end
 gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
 
 gem 'popper_js', '~> 1.11.1'
-gem 'bootstrap', '~> 4.0.0.alpha6'
+gem 'bootstrap', '4.0.0.alpha6'
```

Comment:
You were using a slightly different version of bootstrap that the guides suggest.  This locks it to `Bootstrap 4 Alpha 6` (the `~>` is more permissive of later versions, and because of that you're app used to use `Bootstrap 4 Beta`)